frontend/pages/features/index.js

```diff
@@ -0,0 +1,157 @@
+import React, { useContext } from "react";
+import { Container } from "@chakra-ui/react";
+import RouteButton from "../../src/components/RouteButton";
+import mixpanel from "mixpanel-browser";
+import { getLayout, getLayoutProps } from "../../src/layouts/WideInfoPage";
+import { AWS_ASSETS_PATH } from "../../src/core/constants";
+import { MIXPANEL_EVENTS } from "../../src/core/providers/AnalyticsProvider/constants";
+import FeatureCard from "../../src/components/FeatureCard";
+import useRouter from "../../src/core/hooks/useRouter";
+import UIContext from "../../src/core/providers/UIProvider/context";
+
+const assets = {
+  cryptoTraders: `${AWS_ASSETS_PATH}/crypto+traders.png`,
+  lender: `${AWS_ASSETS_PATH}/lender.png`,
+  DAO: `${AWS_ASSETS_PATH}/DAO .png`,
+  NFT: `${AWS_ASSETS_PATH}/NFT.png`,
+};
+
+const Features = () => {
+  const router = useRouter();
+  const ui = useContext(UIContext);
+
+  const mixpanelReport = function (name, section) {
+    const _report = function () {
+      if (mixpanel.get_distinct_id()) {
+        mixpanel.track(`${MIXPANEL_EVENTS.BUTTON_CLICKED}`, {
+          full_url: router.nextRouter.asPath,
+          buttonName: name,
+          page: `features`,
+          section: section,
+        });
+      }
+    };
+    return _report;
+  };
+
+  return (
+    <Container id="container" maxW="container.xl">
+      {!ui.isMobileView && (
+        <RouteButton
+          variant="orangeAndBlue"
+          onClick={mixpanelReport("Learn More", "main")}
+          href={"/discordleed"}
```

Comment:
Should not be this link to usecases, not a discord?

Reply:
I don't think so. I will ask Daria.